packages/kbn-unified-doc-viewer/src/services/types.ts

```diff
@@ -40,6 +41,7 @@ export interface DocViewRenderProps {
   filter?: DocViewFilterFn;
   onAddColumn?: (columnName: string) => void;
   onRemoveColumn?: (columnName: string) => void;
+  docViewsRegistry?: DocViewsRegistry | ((prevRegistry: DocViewsRegistry) => DocViewsRegistry);
```

Comment:
nit: this whole file is using the term `docView`
I think it would be great if we stick to the same singular naming convention. wdyt ?

Ex - `DocViewRenderFn`, `DocViewFactory`, `DocViewFilterFn`